src/components/Button/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+
+import './style.scss';
+
+const Button = ({
+  buttonClassName,
+  text,
+  disabled,
+  type = 'button',
+}) => (
+  <button
+    className={classnames('button', buttonClassName)}
+    disabled={disabled}
+    // eslint-disable-next-line react/button-has-type
```

Comment:
It's okay to have a type. Why disable the rule?

Reply:
If i don't disable this error is shown: 
`The button type attribute must be specified by a static string or a trivial ternary expression.`